.typos.toml

```diff
@@ -0,0 +1,34 @@
+[default.extend-identifiers]
+AnserStyle = "AnserStyle"
+datas = "datas"
+impl_froms = "impl_froms"
+selfs = "selfs"
+strustS = "strustS"
+
+[default.extend-words]
+anser = "anser"
+ba = "ba"
+foor = "foor"
```

Comment:
Where is this from?

Reply:
![image](https://github.com/rust-lang/rust-analyzer/assets/71162630/4d729df8-baf1-45a5-bc60-c23057d94489)

Reply:
Ah, right. Let's change the macro to `concat!("fo", "o", 0, r#"bar"#, "\n", false, '"', '\0');` so we don't need it any more.